src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
```

Comment:
In both functions you are querying all the nodes from this list, it would be more optimal to search the tree for them once here with querySelectorAll, and grab the ul element only in sortList

Reply:
Also I would target the nodes not by the generic element `li` but rather by the specific data attribute like `[data-salary]`